docs/rules/jsx-max-props-per-line.md

```diff
@@ -58,4 +58,4 @@ The following patterns are not considered warnings:
 
 ## When not to use
 
-If you are not using JSX then you can disable this rule.
+If you are not using JSX or you already use the [`max-len` rule](http://eslint.org/docs/rules/max-len) then you can disable this rule.
```

Comment:
I'm not sure it needs mentioning. Many rules happen to overlap in some cases, but aren't conceptually exclusive rules - this case applies.

I guess "if you are not using jsx, or if you are primarily concerned with line length and the max-len rule is sufficient for you" but I'm not sure what value it adds.